examples/cassandra/README.md

```diff
@@ -350,36 +335,53 @@ UN  10.244.0.5  74.09 KB   256     100.0%            86feda0f-f070-4a5b-bda1-2ee
 UN  10.244.3.3  51.28 KB   256     100.0%            dafe3154-1d67-42e1-ac1d-78e7e80dce2b  rack1
 ```
 
-Now let's scale our cluster to 4 nodes:
+## Step 3: Scale up the Cassandra cluster
+
+Now let's scale our Cassandra cluster to 4 pods.  We do this by telling the
+Replication Controller that we now want 4 replicas.
 
 ```sh
 $ kubectl scale rc cassandra --replicas=4
 ```
 
-In a few moments, you can examine the status again:
+You can see the new pods listed:
 
-```sh
-$ kubectl exec -ti cassandra -- nodetool status
+```console
+$ kubectl get pods -l="app=cassandra" -o wide
+NAME              READY     STATUS    RESTARTS   AGE       NODE
+cassandra-21qyy   1/1       Running   0          6m        kubernetes-minion-b286
+cassandra-81m2l   1/1       Running   0          47s       kubernetes-minion-b286
+cassandra-8qoyp   1/1       Running   0          47s       kubernetes-minion-9ye5
+cassandra-q6sz7   1/1       Running   0          6m        kubernetes-minion-9ye5
+```
+
+In a few moments, you can examine the Cassandra cluster status again, and see
+that the new pods have been detected by the custom `SeedProvider`:
+
+```console
+$ kubectl exec -ti cassandra-xxxxx -- nodetool status
 Datacenter: datacenter1
 =======================
 Status=Up/Down
 |/ State=Normal/Leaving/Joining/Moving
 --  Address     Load       Tokens  Owns (effective)  Host ID                               Rack
-UN  10.244.2.3  57.61 KB   256     49.1%             9d560d8e-dafb-4a88-8e2f-f554379c21c3  rack1
-UN  10.244.1.7  41.1 KB    256     50.2%             68b8cc9c-2b76-44a4-b033-31402a77b839  rack1
-UN  10.244.0.5  74.09 KB   256     49.7%             86feda0f-f070-4a5b-bda1-2eeb0ad08b77  rack1
-UN  10.244.3.3  51.28 KB   256     51.0%             dafe3154-1d67-42e1-ac1d-78e7e80dce2b  rack1
+UN  10.244.0.6  51.67 KB   256     48.9%             d07b23a5-56a1-4b0b-952d-68ab95869163  rack1
+UN  10.244.1.5  84.71 KB   256     50.7%             e060df1f-faa2-470c-923d-ca049b0f3f38  rack1
+UN  10.244.1.6  84.71 KB   256     47.0%             83ca1580-4f3c-4ec5-9b38-75036b7a297f  rack1
+UN  10.244.0.5  68.2 KB    256     53.4%             72ca27e2-c72c-402a-9313-1e4b61c2f839  rack1
 ```
 
-### Using a DaemonSet
+## Step 4: Delete the Replication Controller
 
-Before you start this section, __delete the replication controller__ you created above:
+Before you start Step 5, __delete the replication controller__ you created above:
 
 ```sh
 $ kubectl delete rc cassandra
 ```
 
-In Kubernetes a _[Daemon Set](../../docs/admin/daemons.md)_ can distribute pods
+## Step 5: Use a DaemonSet instead of a Replication Controller
+
+In Kubernetes, a [_Daemon Set_](../../docs/admin/daemons.md) can distribute pods
 onto Kubernetes nodes, one-to-one.  Like a _ReplicationController_, it has a
 selector query which identifies the members of its set.  Unlike a
```

Comment:
In this case, are the members of its set pods or nodes?  What it mean to be a member of a DS?